src/main/java/duke/TaskToDos.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+/**
+ * Represents task of todo type
+ *
+ * @author sibinhho99-nus
+ */
+public class TaskToDos extends Task {
```

Comment:
each instance of this class refers to a single to-do item instead of multiple to-dos; perhaps `TaskToDo` would be a more appropriate name?

Reply:
(same for `TaskDeadlines` and `TaskEvents`)